extensions/OrangeFactor/web/js/orange_factor.js

```diff
@@ -82,8 +85,7 @@ $(function() {
                 getOrangeCount(data);
             })
             .fail(function() {
-                $('#graph-count').hide();
-                $('#orange-graph').hide()
+                $('#orange-count').text('Please sign into OrangeFactor first');
```

Comment:
It would be really stellar if this included a sign-in link, or even just a link to the OF homepage.